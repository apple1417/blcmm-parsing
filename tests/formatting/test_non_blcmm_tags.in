<test>
	<inner />
	<parent>
		<child><BLCMM></child>
	</parent>
</test>

Extra content
