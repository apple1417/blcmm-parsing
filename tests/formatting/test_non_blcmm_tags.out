<test>
	<inner />
	<parent>
		<child>&lt;BLCMM&gt;</child>
	</parent>
</test>
